example/take_screenshot.js

```diff
@@ -0,0 +1,17 @@
+const bin = require('ffmpeg-binaries').ffmpegPath();
+const cp = require('child_process');
+const ytdl = require('..');
+
+const takeScreenshot = (url, outFile, position) => new Promise((resolve, reject) => {
+  ytdl.getInfo(url).then(info => {
+    let format = info.formats[0];
+    cp.exec(`"${bin}" -i "${format.url}"${!format.live && position ? ` -ss ${position}` : ''} -vframes 1 -an -y ${outFile}`, (error, stdout, stderr) => {
```

Comment:
Thoughts on using `exec` vs `execFile`? Using `execFile` would auto quote and escape the filepath, and arguments given.

https://nodejs.org/api/child_process.html#child_process_child_process_execfile_file_args_options_callback

Reply:
never fully understood exec + execFile's use of the arg array...
programms always end up complaining about wrong params or ignoring some